contact.html

```diff
@@ -4,8 +4,37 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web&display=swap" rel="stylesheet">
```

Comment:
I had to give this some thought, due to Amir's comment NOT to use supporting packages.  In this case, you are assuring the fonts you use are available, which is a reasonable thing to do.  even though they are style sheets, they are font specific.